recipes/strainr2/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: strainr2
+  version: {{ version }}
+
+source:
+  url: https://github.com/BisanzLab/StrainR2/archive/v{{ version }}.tar.gz
+  sha256: 6ed9671c875ec6388960dbac02a73629d58ad20adad1fe297d905d8969ec0489
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+  - python=3.12.0
+  - r-base=4.3.2
+  run:
+  - bbmap=39.01
+  - fastp=0.23.4
```

Comment:
Why are all the dependencies strictly pinned? Is it possible to make them a bit more relaxed, i.e `>=`